pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1421,6 +1421,70 @@ func parseIpspec(ipspec *zconfig.Ipspec,
 		config.DhcpRange.Start = start
 		config.DhcpRange.End = end
 	}
+
+	addrCount := types.GetIPAddrCountOnSubnet(config.Subnet)
+	if addrCount < types.MinSubnetSize {
+		if addrCount == 0 {
+			return nil
```

Comment:
In which case you do hit this (is it for a switch NI? in that case I wouldn't expect parseIpspec to be called)

Thus it result to any later failures to allocate IP addresses and do those make sense to a user?